pydicom/util/codify.py

```diff
@@ -33,9 +34,15 @@
 all_cap_re = re.compile('([a-z0-9])([A-Z])')
 
 byte_VRs = [
-    'OB', 'OW', 'OW/OB', 'OW or OB', 'OB or OW', 'US or SS or OW', 'US or SS'
+    'OB', 'OW', 'OW/OB', 'OW or OB', 'OB or OW', 'US or SS or OW', 'US or SS',
+    'OD', 'OL'
 ]
 
+if in_py2:
+    int_type = long
+else:
+    int_type = int
+
 
```

Comment:
maybe move this to `compat.py` where `number_types` if defined?

Reply:
Ah, you caught me - I had the same thought, but then was too lazy to follow through.  I'll do it.